Routing/RedirectRouteProvider.php

```diff
@@ -43,9 +43,10 @@ public function getRouteCollectionForRequest(Request $request)
         // server encodes the url and symfony does not encode it
         // symfony decodes this data here https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/Routing/Matcher/UrlMatcher.php#L91
         $pathInfo = rawurldecode($request->getPathInfo());
+        $host = $request->getHttpHost();
```

Comment:
is there a reason for using `getHttpHost` instead of `getHost`? i suspect that will make problems when running an application on a non-default port